openshift/release/generate-release.sh

```diff
@@ -5,7 +5,9 @@ source $(dirname $0)/resolve.sh
 release=$1
 
 source_output_file="openshift/release/knative-eventing-kafka-source-ci.yaml"
+source_postinstall_output_file="openshift/release/knative-eventing-kafka-source-postinstall-ci.yaml"
 channel_output_file="openshift/release/knative-eventing-kafka-channel-ci.yaml"
+channel_postinstall_file="openshift/release/knative-eventing-kafka-channel-postinstall-ci.yaml"
```

Comment:
Should we apply these two new files [here](https://github.com/openshift-knative/eventing-kafka/blob/release-next/openshift/e2e-common.sh#L128) and [here](https://github.com/openshift-knative/eventing-kafka/blob/release-next/openshift/e2e-common.sh#L143) ?

Reply:
yes, using the post-install is coming in a separate PR. Wanted to make sure this is working first.